### PR TITLE
Add searchable prop in CategoryWidget to optionally use the search functionality

### PR DESCRIPTION
**Pull request by Clebal**
Story details: https://app.shortcut.com/cartoteam/story/210130

image

### CODE REVIEW

**Comment by borja-munoz** [comment]
Sergio, I think we are missing the number of Others categories within parenthesis, as defined by @josegilg

File: packages/react-ui/src/widgets/CategoryWidgetUI.js [hunk 1]
```diff
@@ -363,11 +369,6 @@ function CategoryWidgetUI(props) {
     }
   }, [animation, sortedData]);
 
-  const getCategoriesCount = useCallback(() => {
-    const blocked = blockedCategories.length;
-    return blocked ? data.length - blocked : data.length - maxItems;
-  }, [data, maxItems, blockedCategories]);
-
   // Separated to simplify the widget layout but inside the main component to avoid passing all dependencies
   const CategoryItem = (props) => {
```
Comment (VictorVelarde) [inline packages/react-ui/src/widgets/CategoryWidgetUI.js:373]: Any refactor here, moving 'internal' components to different files would be really appreciated

**Review by VictorVelarde — APPROVED** [review]
👍🏻

File: packages/react-ui/src/widgets/CategoryWidgetUI.js (hunk 1 above)
Comment (Clebal) [reply packages/react-ui/src/widgets/CategoryWidgetUI.js:373]: Great, I'll add it to my backlog 👌

**Pull request merged**